now.json

```diff
@@ -262,6 +262,12 @@
       "headers": {
         "Location": "/blog/google-apps-for-business-disabling-confidential-mode"
       }
+    },
+    {
+      "src": "/*",
+      "headers": {
+        "X-Clacks-Overhead": "GNU Terry Pratchett"
+      }
```

Comment:
I am unfamilair with 'now' , but if this route will match all routes without blocking the specific matches above it, then it looks good. 
Otherwise consider the 'continue' property? https://zeit.co/docs/v2/deployments/routes/#continue

Reply:
Nice! Now I can put it at the top of the stack and add the header to the redirects as well!
(The routing in now is ordered and returns when it first matches *unless you specify `continue`*)